Gemfile

```diff
@@ -2,4 +2,4 @@ ruby "2.5.1"
 
 source 'https://rubygems.org'
 gem 'rspec'
-gem 'rake'
\ No newline at end of file
+gem 'rake'
```

Comment:
You could add a linter specifically for the development environment. Also, move rspec to the development and test environment too